pkgs/servers/soft-serve/default.nix

```diff
@@ -19,9 +19,16 @@ buildGoModule rec {
 
   nativeBuildInputs = [ makeWrapper ];
 
+  passthru.tests = {
+    soft-serve = nixosTests.soft-serve;
+  };
+
   postInstall = ''
     wrapProgram $out/bin/soft \
-      --prefix PATH : "${lib.makeBinPath [ git ]}"
+      --prefix PATH : "${lib.makeBinPath [
+        git
+        bash # For commit hook
```

Comment:
Shouldn't we rather fix the commit hook?

Reply:
Sure, patching the shebang would generally be preferred, but look at how the hook it is defined https://github.com/charmbracelet/soft-serve/blob/7cd0583487cd9573270d2dc0c1700a6839ac1601/server/hooks/hooks.go#L121 

This is not as simple as path-shebangs, but would involve patching or sed'ing of the linked source file.

More importantly, I'm not sure if the hook will then still work after the substituted path is garbage-collected, since the template seems to be used for writing hooks to the data directory / the directory of the server-side repo.

This might also be why git is added in a similar way?

Reply:
then that's actually fine